modules/travel_pay/app/services/travel_pay/service.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module TravelPay
+  class Service
```

Comment:
I think we're almost there. I'd like to have the client simply be a wrapper around Faraday that bundles requests together:

In the client:
```ruby
def get_claims(current_user)
  veis_token = self.request_veis_token
  sts_token = self.request_sts_token(current_user)
  btsss_token = self.request_btsss_token(veis_token, sts_token)

  # Faraday will return the response object or throw a Faraday::Error
  return request_claims(veis_token, btsss_token)
end
```

In the service:

```ruby
def get_claims(current_user)
  claims_response = client.get_claims(current_user)
  
  # do all the business logic-y stuff to the claims response (initial sort, parse, etc)

  return sorted_claims;
end
```

Reply:
What do you think about this? What are the drawbacks?

Reply:
I think this makes sense - with the benefit that adding specs for the service can also test the claims sort logic. The service doesn't do a whole lot right now, so I don't see the drawbacks.